packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/nodejs/integrationsteps.md

```diff
@@ -55,58 +75,108 @@ If the org's application is properly configured with multiple factors, `idx.regi
 
- If the call is successful, the method returns a status of `Idx.Status:PENDING` and a `nextStep` field that requires verification, which indicates that the SDK is ready for the email verification code. The next step is to redirect the user to the email verification code page, as shown in the SDK sample application route to `/enroll-authenticator`.
+ If the call is successful, the method returns a status of `Idx.Status:PENDING` and a `nextStep` field that requires verification, which indicates that the SDK is ready for the email verification code. The next step is to redirect the user to the email verification code page, similar to the following wireframe (see also the SDK sample application route to `/enroll-authenticator`).
 
-<div class="half border">
+<div class="half wireframe-border">
 
-![Displays an Enroll email authenticator page that includes the field for the email verification code. Also displays a Verify button that continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-email-verify-nodejs.png)
+![Email verification code input form](/img/oie-embedded-sdk/wireframes/enter-verification-code-form-g2r5.png)
 
 </div>
 
+<!--
+
+Source image:
+
+https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%8C%9F-Updated-Diagrams-for-Dev-Docs?node-id=2393%3A2128#233281241
+
+Group 2, row 5
+
+-->
+
 ### 6: The user submits the email verification code
 
 The next step is to call `idx.register`, again passing in the verification code. In the email verification use case, the code passed into the method is the code found in the verification email.
 
 Based on the configuration described in [Set up your Okta org for a multifactor use case](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#set-up-your-okta-org-for-a-multifactor-use-case), the app in this use case is set up to require one possession factor (either email or phone). After the email authenticator is verified, the phone authenticator becomes optional. In this step, the `nextStep` field can include `canSkip` for the phone authenticator. You can build a **Skip** button in your form to allow the user to skip the optional phone authenticator.
 
-<div class="half border">
+<div class="half wireframe-border">
 
-![Displays a Select authenticator page that includes a selection for Phone authentication. Also displays a Select button that continues the flow and a Skip button that skips the phone authentication flow. ](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-phone-auth-and-skip-nodejs.png)
+![Choose authenticator form with Phone option and skip button](/img/oie-embedded-sdk/wireframes/choose-authenticator-phone-optional-form-g2r30.png)
 
 </div>
 
+<!--
+
+Source image:
+
+https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%8C%9F-Updated-Diagrams-for-Dev-Docs?node-id=2393%3A2128#233281241
+
+Group 2, row 30
+
+-->
+
```

Comment:
On line 205, under Option 2: The user selects the phone authenticator, you've missed replacing a screenshot for a wireframe.

Reply:
I did have trouble with this one (there is another example somewhere else, too) — the nodejs flow/app design is different enough to the others that I didn't think the wireframes worked for this particular case, so I ended up leaving it as-is. What do you think?